src/main.cpp

```diff
@@ -179,123 +142,136 @@ int main(int argc, char **argv){
             dt = time.toSec() - time_;
             time_ = time.toSec();
         }
+        bool collision = false;
 
-        if (base_ref_) // If there is a twist message in the queue
+        for (std::vector<Robot*>::iterator it = robots.begin(); it != robots.end(); ++it)
```

Comment:
Do we want to check collision for any of the two robots? Or is there a main robot and a NPC (non playable character) robot? The latter case would also avoid having to decide which one to keep in field of the simulation. 

And should the NPC robot in that case also be able to trigger moving objects? I think not.

Reply:
Both robots are the main robots. So they should both trigger moving objects.
Currently the visualisation follows the first robot